felix/bpf-gpl/tc.c

```diff
@@ -108,14 +117,16 @@ static CALI_BPF_INLINE int calico_tc(struct __sk_buff *skb)
 		case CALI_SKB_MARK_BYPASS_FWD:
 			CALI_DEBUG("Packet approved for forward.\n");
 			ctx.fwd.reason = CALI_REASON_BYPASS;
```

Comment:
If we do not deny such packets, why do we set the reason? :thinking: Perhaps we should not set it?